### ISSUE TITLE
Symbology for Vegetation Layers in RSContext

### ISSUE THREAD

**Issue by MattReimer** [Riverscapes/RiverscapesXML#265]
Note to self: The vegetation starts in the RSContext project and is shared by lots of derivatives. This is going to take a manual export from the **Raster Attribute Table (RAT)** in order to work.

@Cashe93 I'm aiming at the following assumptions for layer symbology. Can you correct me if any of them are wrong?

* EVT
    * ***Veg Type - EVT Class***: `Existing_Veg_EVT_Class`: Just keyed off the `EVT_Name` name in the RAT
    * ***Veg Type - EVT Name***: `Existing_Veg_EVT_Name`: Use the `EVT_Name` in the RAT and the and the `R`, `G`, `B` columns for the color.
    * ***Existing Riparian***: `Existing_Veg_Riparian`: This is a boolean operation of `EVT_PHYS === 'Riparian`
* BPS
    * ***Veg Type - BPS Name***: `Historic_Veg_BPS_Name`: Use the `BPS_NAME` field and the `R`, `G`, `B` columns for the color
    * ***Historic Riparian***: `Historic_Veg_Riparian`: This is a boolean operation of `GROUPVEG === 'Riparian` 

More Questions for @Cashe93: 

1. Where does the color from the EVT class come from? Do I need to manually export them from ESRI?
2. What are the exact RGB color values for Existing Riparian? It's some green value for "riparian" and white for "other" right? 
2. There's no `BPS class` equivalent. I'm assuming that's because there's no corresponding column in the RAT?

**Comment by MattReimer** [comment]
iPython python code just FYI:

https://github.com/Riverscapes/RiverscapesXML/blob/evtbps/python/jupyter/veg_rasters/evt_bps.ipynb

**Comment by Cashe93** [comment]
@MattReimer, I'm currently in the field but will have answers to these questions by this evening. Sorry for the delay!

**Comment by MattReimer** [comment]
No worries and no rush. Whenever you have a minute.

**Comment by Cashe93** [comment]
@MattReimer, yes those are the layers. The answers to your questions are as follows:

1. The colors for the vegetation rasters should currently exist as LYR files within the esri shared folder. The .lyr names you are looking for are:

- Existing_Veg_EVT_Class
- Existing_Veg_EVT_Name
- Existing_Veg_Riparian
- Historic_Veg_BPS_Name
- Historic_Veg_Riparian

Is this what you meant by colors?

2. The RGB values for the Riparian are:

- Riparian (Green): R = 56, G = 168, B = 0
- Non-Riparian (White): R = 255, G = 255, B = 255

3. Correct, LANDFIRE doesn't appear to have a BPS - Class attribute

**Comment by MattReimer** [comment]
OK. thanks. that's what I need.
